beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/generator/ExpiredHousekeepingMetadataGenerator.java

```diff
@@ -59,8 +67,17 @@ public ExpiredHousekeepingMetadataGenerator(String cleanupDelay) {
     List<HousekeepingEntity> housekeepingEntities = new ArrayList<>();
 
     switch (listenerEvent.getEventType()) {
+    case CREATE_TABLE:
+      housekeepingEntities.add(generateHousekeepingEntity((CreateTableEvent) listenerEvent, clientId));
+      break;
     case ALTER_TABLE:
-      housekeepingEntities.add(generateHousekeepingEntity(listenerEvent, clientId));
+      housekeepingEntities.add(generateHousekeepingEntity((AlterTableEvent) listenerEvent, clientId));
```

Comment:
this and alter_partition will create a duplicate entry right? You want to add alter because of a location change to capture that correct? But if there is no location change do we need a duplicate entry?

Reply:
There won't be a duplicate entry because we do a look up in the scheduler to find the existing record and update that. Although it will make a new entry if the table or partition name has changed but we need to change around of lot of things to handle that case.

https://github.com/ExpediaGroup/beekeeper/blob/ae0ae70e9a1f3310cb49cb22669d7f8369ff4cad/beekeeper-scheduler/src/main/java/com/expediagroup/beekeeper/scheduler/service/ExpiredHousekeepingMetadataSchedulerService.java#L68

Reply:
I think a new entry if the table is changes makes perfect sense. We can keep the old entry it will just have nothing to delete as the table is gone already.
So this caputures alters on the cleanupDelay property only right?

Reply:
Yeah exactly

Reply:
ok let's go with this for now